pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -1190,3 +1190,13 @@ def test_tz_localize_invalidates_freq():
     dti2 = dti[:1]
     result = dti2.tz_localize("US/Eastern")
     assert result.freq == "H"
+
+
+def test_replace_preserves_fold():
```

Comment:
should go here: pandas/tests/scalar/timestamp/test_unary_ops.py (there is another replace test)

Reply:
Done.